packages/govuk-frontend/src/govuk/components/character-count/character-count.test.js

```diff
@@ -1,24 +1,16 @@
 const { goToComponent, renderAndInitialise } = require('govuk-frontend-helpers/puppeteer')
 const { getExamples } = require('govuk-frontend-lib/files')
 
-const jestPuppeteerConfig = require('../../../../../../jest-puppeteer.config')
-
-// Detect when browser has been launched headless
-const { headless = 'new' } = jestPuppeteerConfig.launch
-
-// The longest possible time from a keyboard user ending input and the screen
-// reader counter being updated: handleFocus interval time + last input wait time
-// but raised to the nearest whole second when browser is not headless as timers
-// in background pages will be aligned, clamped or throttled
-const debouncedWaitTime = headless ? 1500 : 2000
-
-// When headless, keydown-to-keyup time appears to affect event throttling,
-// affecting `lastInputTimestamp` and screen reader status message updates
-const keyupWaitTime = headless ? 0 : 50
-
 describe('Character count', () => {
   let examples
 
+  const focusIntervalTime = 1000 // ms
+  const lastInputOffsetTime = 500 // ms
+
+  // The longest possible time from a keyboard user ending input and the screen
+  // reader counter being updated (+ 50ms to stay outside the total wait time)
+  const debouncedWaitTime = focusIntervalTime + lastInputOffsetTime + 50
```

Comment:
Since https://github.com/alphagov/govuk-frontend/pull/3543 (or perhaps in recent Puppeteer updates) we're seeing much more accurate timers

Running assertions at the exact `setTimeout()` period can fail the test so I've added 50ms on